composer.json

```diff
@@ -39,7 +39,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "3.1-dev"
+            "dev-master": "3.3-dev"
```

Comment:
Sorry for the late reply. In the past, we've created a branch to support stable releases while making master the development branch for the next minor or major version, depending on the changes.